examples/Distributed-example/readme.md

```diff
@@ -0,0 +1,385 @@
+## About OpenPai
+OpnePai's user manual is here. Through OpenPai, you can easily call on powerful computing power to train your complex models:
+https://openpai.readthedocs.io/en/latest/manual/cluster-user/advanced-jobs.html
+
+You need to submit your program to OpenPai in the form of Job. OpenPai uses container technology to provide a rich development environment. You can visit the following link to learn how to submit Job and learning the concept of Task.
+https://openpai.readthedocs.io/en/latest/manual/cluster-user/how-to-use-advanced-job-settings.html#multiple-task-roles
+
+In this document you can learn how different tasks communicate in Pai.
+https://openpai.readthedocs.io/en/latest/manual/cluster-user/how-to-use-advanced-job-settings.html#environmental-variables-and-port-reservation
+
+Job Exit Spec, Retry Policy, and Completion Policy
+https://openpai.readthedocs.io/en/latest/manual/cluster-user/how-to-use-advanced-job-settings.html#job-exit-spec-retry-policy-and-completion-policy
+## Why distributed data parallel?
+Pytorch has two ways to split models and data across multiple GPUs: nn.DataParallel and nn.DistributedDataParallel. nn.DataParallel is easier to use (just wrap the model and run your training script). However, because it uses one process to compute the model weights and then distribute them to each GPU during each batch, networking quickly becomes a bottle-neck and GPU utilization is often very low. Furthermore, nn.DataParallel requires that all the GPUs be on the same node and doesn’t work with Apex for mixed-precision training.
+
+
+
+## A tutorial on writing distributed applications using Pytorch
+The tutorial on writing distributed applications in Pytorch has much more detail than necessary for a first pass and is not accessible to somebody without a strong background on multiprocessing in Python. It spends a lot of time replicating the functionality in nn.DistributedDataParallel. However, it doesn’t give a high-level overview of what it does and provides no insight on how to use it. (https://pytorch.org/tutorials/intermediate/ddp_tutorial.html)
+
+The closest to a MWE example Pytorch provides is the Imagenet training example（https://github.com/pytorch/examples/tree/master/imagenet）   . Unfortunately, that example also demonstrates pretty much every other feature Pytorch has, so it’s difficult to pick out what pertains to distributed, multi-GPU training.
+
+
+
+
+
+## Outline
+This tutorial is really directed at people who are already familiar with training neural network models in Pytorch, and I won’t go over any of those parts of the code. I’ll begin by summarizing the big picture. I then show a minimum working example of training on Cifar10 using on GPU. I modify this example to train on multiple GPUs, possibly across multiple nodes, and explain the changes line by line. Importantly, I also explain how to run the code. As a bonus, I also demonstrate how to use Apex to do easy mixed-precision distribued training.
```

Comment:
I think we can this paragraph. The instructure of the titles can tell the users about the document structure.